lib/usb-device.js

```diff
@@ -16,7 +16,7 @@ class USBDevice {
   }
 
   // opens the device and sets up the endpoints for out object
-  async claimDevice () {
+  claimDevice () {
```

Comment:
Good to know. I'll do a read through of all my public facing functions and make sure things are consistent. Once I get testing up to an acceptable level my next todo is documenting all public facing functions for people who want to use the library, as well as internal functions that would need to be implemented for a TCP Device class. 

I'll probably end up writing more documentation than anyone will ever need :)